fmn/consumer/consumer.py

```diff
@@ -212,7 +169,11 @@ def work(self, session, raw_msg):
                 )
                 session.add(user)
                 session.commit()
-                notify_prefs_change(openid)
+                _log.info('Broadcasting message to Celery workers to update cache for %s', openid)
+                find_recipients.apply_async(
+                    ({'topic': 'fmn.internal.refresh_cache', 'body': openid},),
+                    exchange='fmn.tasks.reload_cache'
```

Comment:
This string gets used in several places - might make sense to make it a "constant".